js/main.js

```diff
@@ -2,13 +2,29 @@ var searchForm = document.querySelector('.search-form');
 var searchBar = document.querySelector('#game-search')
 var logosRow = document.querySelector('.logo-images');
 var homePageText = document.querySelector('.main-text');
-var searchResults = document.querySelector('.search-results')
+var searchResults = document.querySelector('.search-results');
+var storeListings = document.querySelector('.store-listings-results')
+
+var storesList = [];
 
 
 searchForm.addEventListener('submit', submitAction);
 
+function stores() {
+  var stores = new XMLHttpRequest();
```

Comment:
You should change the name of this variable, you don't want variables with the same name as the function they are in